### DISCUSSION TITLE
Can't login

### DISCUSSION THREAD

**Discussion by pamelaAHeredia** [julian-poidevin/MBPMid2010_GPUFix#148] [Q&A]
Hi! I just installed the patch on my MBP and looked like a suscessfully installation. I restarted the machine and now i can't login. What happened? How can I fix it? I need to work...

**Comment by julian-poidevin** [comment]
Hello, the fix is not changing the way the MacBook behaves during log in. Can you be more specific? Is your password being rejected ?
